foundations/installations/installations.md

```diff
@@ -298,9 +298,8 @@ wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
 
 - Enter the following command in your terminal to install **Google Chrome** `.deb` package
 
-       ~~~bash
        sudo apt install ./google-chrome-stable_current_amd64.deb
-       ~~~
+       
```

Comment:
If you use the markdown preview tool that is listed in the checklist, you can see the effects of your changes. 
https://www.theodinproject.com/lessons/preview

I believe the intention is to have it in a code block, which this change removes. The issue with the code block is that there were too many spaces before this line.

```suggestion
    ```bash
    sudo apt install ./google-chrome-stable_current_amd64.deb
    ```
```